demo/kubernetes/rook-operator-instance.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rook.io/v1beta1
+kind: Cluster
```

Comment:
Note that the kind is unique to the rook.io namespace. If the kind is `RookCluster`, the TPR will look a bit redundant like below. What if we call it `Storage` or keep it as `Cluster`?
```
NAME                   DESCRIPTION             VERSION(S)
rook-cluster.rook.io   Managed rook clusters   v1beta1
```